web/src/main/java/au/org/emii/classifier/LabelClassifier.java

```diff
@@ -29,8 +29,8 @@ public LabelClassifier(ThesaurusFinder thesaurusFinder, String vocabularyScheme,
         List<AodnTerm> matchingTerms = vocabularyThesaurus.getTermWithLabel(value);
 
         if (matchingTerms.isEmpty()) {
-            logger.warn(String.format("Could not find term with label='%s' in vocabulary='%s'",
-                value, vocabularyThesaurus.getThesaurusTitle()));
+            logger.warn(String.format("Could not find term with label='%s' in vocabulary='%s' discoveryParams='%s=%s'",
+                    value, vocabularyThesaurus.getThesaurusTitle(), "orgUnit", value));
```

Comment:
CAn we make this one configured as well.  Otherwise we have to change this to  reuse the labelclassifier for another facet